tests/unit/broccoli/default-packager/ember-cli-internal-test.js

```diff
@@ -193,8 +193,8 @@ var runningTests = false;`);
         addons: [
           {
             contentFor(type) {
-              if (type === 'app-prefix') {
-                return 'CUSTOM APP PREFIX CODE';
+              if (type === 'test-support-prefix') {
```

Comment:
Otherwise, this tests breaks the `Deprecations Broken` job.

Reply:
Is the app prefix stuff still tested somewhere?